containers/Loans/Loans.tsx

```diff
@@ -40,7 +38,7 @@ function Container() {
 		collateralManagerContract,
 		exchangeRatesContract,
 	] = useMemo(() => {
-		if (!(isAppReady && synthetix.js && signer)) return [null, null, null, null, null, null];
+		if (!(isAppReady && synthetixjs && signer)) return [null, null, null, null, null, null];
```

Comment:
I guess it's not from you but god this [null, null, null....] is so ugly

Reply:
lol